packages/design-tokens/design-token-previews/components/createBox.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import includes from "lodash/includes";
+import replace from "lodash/replace";
+import { TokenEntry } from "../types/TokenEntry";
+import { Box } from "../../../components/src/primitives/Box";
+
+export const createBox =
+  (tokenAttribute: string, defaultProps = {}, text = "\0") =>
```

Comment:
The `text` argument is just an exception for font preview. Shouldn't we use a single front preview? Or a `createFrontPreview` function? I left a similar suggestion in another file.